### DISCUSSION TITLE
Security Deputies and Cadets

### DISCUSSION THREAD

**Discussion by Humonitarian** [unitystation/unitystation#7919] [Ideas]
### Deputies:
-Deputies would be a job that can be given through the HoP console, and are not avaliable at start. Deputies have basic security access, but cannot enter the equipment room or open cells, unlike regular Security Officers. The main purpose of Deputies is to allow quick recruitment of assitants into security to increase headcount against a larger threat, such as Nukies or rebels.
Lorewise this would be especially beneficial, as to become a regular Security Officer, one would need an extensive background-check, while Deputies can be recruited on the spot in an emergency.

### Cadets:
-Cadets would be Security-in-Training, avaliable at roundstart. They would start without a tazer, and would be assigned to a Department to man the local Security Office. Cadets would be responsible for watching over departments, and helping spot criminals, as well as assist Security Officers.
This position would be ideal for new players who want to learn how to play Security without immediately having the responsibilities of one.
Cadets would start with a security belt (with a stunbaton and handcuffs), as well as a regular armour vest, and a beret.

PS:
In preferences, a Cadet department preference should be avaliable, where players can choose a department they would like to work in if they become a Cadet. When spawning as a Cadet, they would receive a beret in the colour of that department, as well as department-coloured armbands, basic access to the department, and an encryption key.

**Comment by AutumnNova** [comment]
beret in the color of the department they are looking after*

**Comment by PetMudstone** [comment]
Relevant pull request from Bee: https://github.com/BeeStation/BeeStation-Hornet/pull/1427

I would also like to note that Fulpstation has "deputies", but they act more like the cadets mentioned before instead.
